torch/testing/_internal/common_methods_invocations.py

```diff
@@ -2272,6 +2272,29 @@ def generator():
 
     return list(generator())
 
+def sample_inputs_max_pool2d(op_info, device, dtype, requires_grad, **kwargs):
+    make_arg = partial(make_tensor, device=device, dtype=dtype, requires_grad=requires_grad)
+
+    kerneli = [[3, 2], [3, 3]]
+    stridei = [[2, 2]]
+    Ni = [8, 16]
```

Comment:
Nit: Would be nice to decrease the batch size here, making the test slightly faster I guess.